pkgs/misc/meson64-tools/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchFromGitHub, buildPackages }:
+
+stdenv.mkDerivation rec {
+  pname = "meson64-tools";
+  version = "unstable-2020-08-03";
+
+  src = fetchFromGitHub {
+    owner = "angerman";
+    repo = pname;
+    rev = "a2d57d11fd8b4242b903c10dca9d25f7f99d8ff0";
+    sha256 = "1487cr7sv34yry8f0chaj6s2g3736dzq0aqw239ahdy30yg7hb2v";
+  };
+
+  buildInputs = with buildPackages; [ openssl bison yacc flex bc python3 python2 ];
+
+  preBuild = ''
+    patchShebangs .
+  '';
+
+  makeFlags = [ "PREFIX=$(out)/bin" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/angerman/meson64-tools";
+    description = "Tools for Amlogic Meson ARM64 platforms";
+    maintainers = with maintainers; [ aarapov ];
```

Comment:
Missing license.

Reply:
There is no license specified. Relevant [issue](https://github.com/angerman/meson64-tools/issues/2) in meson64-tools repository is opened. As long as we don't know - it is **unfree**.